package-lock.json

```diff
@@ -4462,6 +4462,35 @@
         "compare-versions": "^3.5.1",
         "fs-extra": "^10.0.0",
         "ts-essentials": "^8.0.0"
+      },
+      "dependencies": {
+        "fs-extra": {
+          "version": "10.0.0",
```

Comment:
strange that there is a version 10.0.0 for fs-extra here.

Reply:
some test dependencies are using it

```
$ npm ls fs-extra

vscode-apache-camel@0.1.3
├── fs-extra@9.1.0
├─┬ vscode-extension-tester@4.2.3
│ ├── fs-extra@10.0.0
│ └─┬ monaco-page-objects@1.9.0
│   └── fs-extra@10.0.0
├─┬ vscode-extension-tester-native@3.0.2
│ ├─┬ @nut-tree/nut-js@1.5.0
│ │ └─┬ @nut-tree/libnut@2.1.0
│ │   └─┬ @nut-tree/libnut-darwin@2.1.0
│ │     └─┬ cmake-js@6.1.0
│ │       └── fs-extra@5.0.0
│ └── fs-extra@9.1.0
└─┬ vscode-uitests-tooling@2.2.0
  └── fs-extra@8.1.0
```